packages/core/stories/components/Footer/Footer.data.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  items: [
+    {
+      heading: 'Footer Primary Links',
+      id: 'FooterLinks1',
+      links: [
+        {
+          href: 'https://mass.gov/topics/living',
```

Comment:
@clairesunstudio Could we change these links to point to https://www.mass.gov? At the moment, these links will redirect to www, which at the moment could trigger some warnings in the browser.